app/javascript/src/Settings/components/Common/TextInputGroup.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import * as React from 'react'
+import { useState } from 'react'
+import { FormGroup, TextInput } from '@patternfly/react-core'
+import type { FormSettingsProps } from 'Settings/types'
+
+const TextInputGroup = (
+  {
+    defaultValue,
+    placeholder,
+    label,
+    name,
+    hint,
+    value,
+    isDefaultValue = false,
+    readOnly = false,
+    inputType = 'text'
+  }: FormSettingsProps) => {
+  const [ inputValue, setValue ] = useState(value)
```

Comment:
```suggestion
  const [value, setValue] = useState(value)
```
or
```suggestion
  const [inputValue, setInputValue] = useState(value)
```

Reply:
@didierofrivia I've seem a couple of these in the PR, make this comment count for all of them :P your call to follow naming convention or not.

Reply:
You are more than right! :)